www/src/css/components/_card-grid.scss

```diff
@@ -0,0 +1,105 @@
+.card-grid {
+  display: grid;
+  grid-column-gap: 15px;
+  grid-row-gap: 15px;
+  grid-auto-flow: dense;
+}
+
+.card-grid-3 {
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+.card-grid-4 {
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+}
+.discord-banner {
+  grid-column: 1 / -1;
+  border: 1px solid #2e2077;
+  background-color: #545eec;
+  display: flex;
+  align-items: center;
+  font-size: 16px;
+  color: white;
+  font-weight: 500;
+  padding: 1.25rem;
+  margin: 1.5rem 0 1rem;
+  background: #545eec;
+  box-shadow: 6px 6px 20px #4750c966, -6px -6px 20px #616cff66;
+}
+.discord-banner > * {
+  display: block;
+}
+
+@media (max-width: 700px) {
```

Comment:
Also not a recommendation for change, but a really neat helper: in Sass you can nest the media query under the original class. It works the same way, but I find that being able to edit base styles + responsive styles in the same block is easier to read and easier to modify.